lib/whatsapp_sdk/version.rb

```diff
@@ -2,5 +2,5 @@
 # typed: strict
 
 module WhatsappSdk
-  VERSION = "0.7.0"
+  VERSION = "0.7.1"
```

Comment:
Let's bump the version in a separate PR.

Reply:
Got it